### ISSUE TITLE
Exception in `revoke_member_privilege`

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-ch#451]
```
11/10/2015 08:44:47:INFO    :chapi:SA: a...@geni.net: Changed member N. R... to role LEAD in project scaleTestSE
11/10/2015 08:44:47:INFO    :chapi:SA: a...@geni.net: Changed lead for project scaleTestSE from urn:publicid:IDN+ch.geni.net+user+s... to urn:publicid:IDN+ch.geni.net+user+n...
11/10/2015 08:44:47:ERROR   :amsoil:[mav1] [SERVER] SERVER_ERROR (list index out of range)
11/10/2015 08:44:47:ERROR   :amsoil:[mav1]   File "/usr/share/geni-ch/chapi/AMsoil-gpo-0.3.3/src/plugins/chapiv1rpc/chapi/MemberAuthority.py", line 484, in revoke_member_privilege
    mc._session)

  File "/usr/share/geni-ch/chapi/AMsoil-gpo-0.3.3/src/plugins/marm/MAv1Implementation.py", line 1327, in revoke_member_privilege
    if rows[0][0] == 'true':

11/10/2015 08:44:47:INFO    :chapi:MA: a...@geni.net: Result from revoke_member_privilege: {'output': '[SERVER] SERVER_ERROR (list index out of range)', 'code': 101, 'value': None}
```

**Comment by ahelsing** [comment]
The query for a project lead fails to consider the case where the query returns 0 rows

**Comment by ahelsing** [comment]
closed via #452
